staging/src/k8s.io/apiserver/pkg/storage/feature/feature_support_checker.go

```diff
@@ -82,26 +86,56 @@ func (f *defaultFeatureSupportChecker) Supports(feature storage.Feature) (bool,
 func (f *defaultFeatureSupportChecker) CheckClient(ctx context.Context, c client, feature storage.Feature) error {
 	switch feature {
 	case storage.RequestWatchProgress:
-		return f.clientSupportsRequestWatchProgress(ctx, c)
+		return f.checkClient(ctx, c)
 	default:
 		return fmt.Errorf("feature %q is not implemented in DefaultFeatureSupportChecker", feature)
 
 	}
 }
 
-func (f *defaultFeatureSupportChecker) clientSupportsRequestWatchProgress(ctx context.Context, c client) error {
+func (f *defaultFeatureSupportChecker) checkClient(ctx context.Context, c client) error {
```

Comment:
I don't see error returned anywhere - let's just not return anythin in this case.

Reply:
Assuming that, we don't need to return an error in other places too (including changing the syntax of etcd3.New).

Reply:
So in that case will remove `error` from `CheckClient` too, because it means the same error as `Supports`

Reply:
Done